### PR TITLE
Issue/8 probes form

### LINKED ISSUE #8: Create Probes Form
Probes

### PR DESCRIPTION
**Pull request by dennypradipta**
This PR will hopefully fix #8

### CODE REVIEW

File: pages/advanced.tsx [hunk 1]
```diff
@@ -1,32 +1,64 @@
...
   const { notificationData, handleAddNotification } = useContext(
     NotificationContext
   );
+  const { probeData, handleAddProbe } = useContext(ProbeContext);
   const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
 
-  const renderProbe = () => {
-    return <div>Hello Probe</div>;
+  type Menu = 'Probe' | 'Notification';
+  const [activeMenu, setActiveMenu] = useState<Menu>('Probe');
```
Comment (kevinhermawan) [inline pages/advanced.tsx:20]: Why we need to move `activeMenu` from `sidebar` to this component?

Comment (dennypradipta) [reply pages/advanced.tsx]: Could you please check again as I have updated the sidebar component to be more reusable :grinning:

**Comment by nicnocquee** [comment]
In the Request block, 

- [ ] add another option to the Body: No Body
- [ ] By default, No Body is selected
- [ ] When there is no body, no need to validate the json

**Comment by nicnocquee** [comment]
By default, in the Notify on section:
- [ ] status code not 2xx is selected
- [ ] response time is longer than is true
- [ ] response time by default should be 2000 ms

**Comment by dennypradipta** [comment]
> In the Request block,
> 
> * [x]  add another option to the Body: No Body
> * [x]  By default, No Body is selected
> * [x]  When there is no body, no need to validate the json

Done

> By default, in the Notify on section:
> 
> * [x]  status code not 2xx is selected
> * [x]  response time is longer than is true
> * [x]  response time by default should be 2000 ms

Done

File: contexts/probe-context.tsx [hunk 2]
```diff
@@ -1,45 +1,476 @@
...
   };
 
+  const handleUpdateProbeRequestEnableBody = (
+    probeId: string,
+    requestIndex: number,
+    value: string
+  ) => {
+    const foundProbe = probes.find((data) => data.id === probeId);
+
+    console.log(value);
```
Comment (nicnocquee) [inline contexts/probe-context.tsx:254]: Remove console.log

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Review by nicnocquee — APPROVED** [review]

**Review by haricnugraha — APPROVED** [review]

**Pull request merged**